backend/app/src/users/users.controller.ts

```diff
@@ -17,6 +17,27 @@ export class UsersController {
     return await this.usersService.getUserById(id);
   }
 
+  @Patch(':id')
+  async updateUser(
+    @Param('id') id: string,
+    @Body('firstName') firstName: string,
+    @Body('lastName') lastName: string,
+    @Body('email') email: string,
+    @Body('password') password: string,
+
```

Comment:
Not in the scope of this ticket, but perhaps for the password, there will need to a be unique process for a user?